utils/rpms/daos.spec

```diff
@@ -257,6 +258,7 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 
 %files server
 %config(noreplace) %{conf_dir}/daos_server.yml
+%attr(0660,root,daos_admins) %{conf_dir}/daos_server.yml
```

Comment:
Are there secrets in this file requiring the `660` permissions?

You are going to need to fix the _Test CentOS 7 RPMs" stage if so:

```
01:14:43.696  + cat /etc/daos/daos_server.yml
01:14:43.696  cat: /etc/daos/daos_server.yml: Permission denied
```

Reply:
Yeah, I think this should be 0664 (world-readable). There's nothing secret in the server config.